tests/data/select-tests.js

```diff
@@ -185,8 +187,13 @@ test('duplicates - single - same id on select triggers change',
   );
 
   assert.ok(
+    inputTriggered,
+    'The input event should be triggered'
+  );
+
+  assert.ok(
     changeTriggered,
-    'The change event should be triggered'
+    'The change event should be triggered after the input event'
```

Comment:
I'm not sure how I feel about enforcing the order of the events within past tests. I can understand adding tests to check that the `input` event is triggered, and adding tests to ensure that one event is caught before the other, but I don't think that requires you to modify old `change`-only tests to cover both those cases.